README.md

```diff
@@ -90,7 +90,7 @@ git checkout -b fix_out_of_date_patch origin/1.22
 ```
  Then when you get to this pull request screen change the base branch from `master` to `1.22`
 
-- By creating a pull request, the PR is entered into the [backlog](https://waffle.io/crosstool-ng/crosstool-ng). A [travis-ci](https://travis-ci.org/crosstool-ng/crosstool-ng/builds) job will run to test your changes against a select set of samples. As they start to get worked, they should be placed in the `Ready` state. PRs that are being worked are `In Progress`. If a questions come up about the commit that might envolve changes to the commit then the PR is placed in `Waiting For Response`, you have two options:
+- By creating a pull request, the PR is entered into the [backlog](https://waffle.io/crosstool-ng/crosstool-ng). A [travis-ci](https://travis-ci.org/crosstool-ng/crosstool-ng/builds) job will run to test your changes against a select set of samples. As they start to get worked, they should be placed in the `Ready` state. PRs that are being worked are `In Progress`. If a questions come up about the commit that might involve changes to the commit then the PR is placed in `Waiting For Response`, you have two options:
```

Comment:
This change should be in a separate commit as it is unrelated to the dtc change

Reply:
@cpackham I've rebased on master and removed this change. Will open a separate PR for the README change.

Reply:
Opened https://github.com/crosstool-ng/crosstool-ng/pull/1445